modules/sdk-coin-bsc/test/unit/transactionBuilder/send.ts

```diff
@@ -0,0 +1,160 @@
+      txBuilder.type(TransactionType.SingleSigSend);
+      txBuilder.value('1000');
+      txBuilder.contract('0xA2959D3F95eAe5dC7D70144Ce1b73b403b7EB6E0');
+      txBuilder.counter(1);
+      txBuilder.fee({
+        fee: '20',
+        gasLimit: '30',
+      });
+      const unsignedTransaction = await txBuilder.build();
+      serializedTransaction = Buffer.from(unsignedTransaction.toBroadcastFormat());
+    });
+
+    it('A and B signing', () => {
+      const B_sign_share = MPC.signShare(B_combine.xShare, B_combine.yShares[1]);
+
+      const A_sign_convert = MPC.signConvert({
+        xShare: A_combine.xShare,
+        yShare: A_combine.yShares[2],
+        kShare: B_sign_share.kShare,
+      });
+
+      const B_sign_convert = MPC.signConvert({
+        wShare: B_sign_share.wShare,
+        aShare: A_sign_convert.aShare,
+      });
+
+      const A_second_sign_convert = MPC.signConvert({
+        bShare: A_sign_convert.bShare,
+        muShare: B_sign_convert.muShare,
+      });
+
+      const A_sign_combine = MPC.signCombine({
+        gShare: A_second_sign_convert.gShare!,
+        signIndex: {
+          i: 1,
+          j: 2,
+        },
+      });
+
+      const B_sign_combine = MPC.signCombine({
+        gShare: B_sign_convert.gShare!,
+        signIndex: {
+          i: 2,
+          j: 1,
+        },
+      });
+
+      const A_sign = MPC.sign(serializedTransaction, A_sign_combine.oShare, B_sign_combine.dShare);
+      const B_sign = MPC.sign(serializedTransaction, B_sign_combine.oShare, A_sign_combine.dShare);
+      const signature = MPC.constructSignature([A_sign, B_sign]);
+
+      MPC.verify(serializedTransaction, signature).should.be.true;
+    });
+
+    it('A and C signing', () => {
+      const C_sign_share = MPC.signShare(C_combine.xShare, C_combine.yShares[1]);
+
+      const A_sign_convert = MPC.signConvert({
+        xShare: A_combine.xShare,
+        yShare: A_combine.yShares[3],
+        kShare: C_sign_share.kShare,
+      });
+
+      const C_sign_convert = MPC.signConvert({
+        wShare: C_sign_share.wShare,
+        aShare: A_sign_convert.aShare,
+      });
+
+      const A_second_sign_convert = MPC.signConvert({
+        bShare: A_sign_convert.bShare,
+        muShare: C_sign_convert.muShare,
+      });
+
+      const A_sign_combine = MPC.signCombine({
+        gShare: A_second_sign_convert.gShare!,
+        signIndex: {
+          i: 1,
+          j: 3,
+        },
+      });
+
+      const C_sign_combine = MPC.signCombine({
+        gShare: C_sign_convert.gShare!,
+        signIndex: {
+          i: 3,
+          j: 1,
+        },
+      });
+
+      const A_sign = MPC.sign(serializedTransaction, A_sign_combine.oShare, C_sign_combine.dShare);
+      const C_sign = MPC.sign(serializedTransaction, C_sign_combine.oShare, A_sign_combine.dShare);
+      const signature = MPC.constructSignature([A_sign, C_sign]);
+
+      MPC.verify(serializedTransaction, signature).should.be.true;
```

Comment:
Do I need to do anything more that just verify?

Reply:
i think this is good - @olibeyene to confirm?

Reply:
The reason I asked is because all the test is doing right now is creating a transaction and signing it with TSS. What we are testing is essentially just that the ECDSA works.

Reply:
yea ultimately it'd be great to verify that the signature deserializes to a signer with the correct pubkey, but this is fine for now

Reply:
this looks good to me